### PR TITLE
user manuals: add slo config doc

### PR DESCRIPTION
**Pull request by saintube**
- Add the user manual of SLO Configuration (both the English and the Chinese ver.).
- Adjust the order of user manuals.

### CODE REVIEW

File: docs/user-manuals/slo-config.md [hunk 1]
```diff
@@ -0,0 +1,408 @@
+# SLO Configuration
+
+## Introduction
+
+Koordinator uses a ConfigMap to manage the SLO configurations. The ConfigMap is stored in`${ConfigNamespace}/${SLOCtrlConfigMap}`
+(`koordinator-system/slo-controller-config` by default), with the following keys respectively:
+
```
Comment (zwzhang0107) [inline docs/user-manuals/slo-config.md:7]: configmap is for slo-controller, which can specify ns/name via startup args

File: docs/user-manuals/slo-config.md [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+      "clusterStrategy": {
+        "enable": true,
+        "cpuSuppressThresholdPercent": 60,
+        "cpuSuppressPolicy": "cpuset",
+        "memoryEvictThresholdPercent": 60
+      }
+    }
+```
+
+3. Verify if the NodeSLO is successfully dispatched.
```
Comment (zwzhang0107) [inline docs/user-manuals/slo-config.md:392]: default value will be omitted in NodeSLO

File: docs/user-manuals/slo-config.md (hunk 1 above)
Comment (saintube) [reply docs/user-manuals/slo-config.md:8]: fixed

File: docs/user-manuals/slo-config.md (hunk 2 above)
Comment (saintube) [reply docs/user-manuals/slo-config.md:393]: added below

**Review by hormes — APPROVED** [review]

**Pull request merged**